packages/gp2-components/src/organisms/NavigationHeader.tsx

```diff
@@ -79,7 +76,15 @@ const NavigationHeader: React.FC<NavigationHeaderProps> = ({
       <div css={[menuButtonStyles]}>
         <MenuButton open={menuShown} onClick={() => onToggleMenu(!menuShown)} />
       </div>
-      <HeaderLogo />
```

Comment:
why did we stop using the HeaderLogo? can't we apply the changes in the component?

Reply:
As far as the new designs for the NavBar, we don't need to use the gp2Logo anymore